libs/datamodel/connectors/dml/src/relation_info.rs

```diff
@@ -11,16 +14,18 @@ pub struct RelationInfo {
     pub name: String,
     /// A strategy indicating what happens when
     /// a related node is deleted.
-    pub on_delete: OnDeleteStrategy,
+    pub on_delete: Option<ReferentialAction>,
+    /// A strategy indicating what happens when
+    /// a related node is deleted.
+    pub on_update: Option<ReferentialAction>,
+    /// Set true if referential actions feature is not in use.
+    pub legacy_referential_actions: bool,
```

Comment:
Is this used only for rendering? We should mention this if it is.

Reply:
No, it's about re-introspection when we overwrite the data model due to comparing relation infos and whatever comes from the DB is different than what we have in the PSL. Ugh, yeah, I know, but...

Reply:
fixing the comment still.